### PR TITLE
force:analytics updates

### PR DESCRIPTION
**Pull request by smithgp**
### What does this PR do?
- remove undocumented `storedQueries` field from sample template-info.json
- format sample template files to match default prettier format
- add oclif topic description for force:analytics and force:analytics:template

### What issues does this PR fix or reference?
- W-7258830 (storedQueries)
- W-7222866 (topic description)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,7 +65,17 @@
     "bin": "sfdx",
     "devPlugins": [
       "@oclif/plugin-help"
-    ]
+    ],
+    "topics": {
```
Comment (lcampos) [inline package.json]: Outside of this change to package.json the rest seems to be formatting changes to the template. Is this correct ?

Comment (smithgp) [reply package.json]: @lcampos Actually, the first commit does remove the `storedQueries: []` field from the template-info.json. The 2nd commit is all formatting of the template files, plus the updates to the unit test to account for the difference in spacing.

Comment (smithgp) [reply package.json]: You use the Hide whitespace changes checkbox in the gear icon on Files changed tab to see the real changes better.

Comment (lcampos) [reply package.json]: Cool, I missed that `storedQueries` change since I was going through this looking at all the changes at once.

**Pull request merged**